src/core/meshcop/border_agent.cpp

```diff
@@ -408,7 +408,7 @@ Error BorderAgent::MeshCoPTxtEncoder::EncodeTxtData(void)
 #if OPENTHREAD_FTD && OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE
     SuccessOrExit(error = AppendBbrTxtEntry(state));
 #endif
-#if OTBR_ENABLE_BORDER_ROUTING
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE
```

Comment:
I remember we used to have (or may still have) a Github Action CI build check that would report an error or warning if the code used an undefined macro. I'm not sure if we still have it, or if for some reason this file is not included in that build check?

I'm just mentioning this as something to check later, since I think it can be helpful (there is always a chance of a typo when adding `#if` checks). So it would be good to have a CI check that would catch such issues.

Reply:
Submitted #11418 for it.